modules/platform/nuxeo-apidoc-server/nuxeo-apidoc-webengine/src/main/java/org/nuxeo/apidoc/browse/Distribution.java

```diff
@@ -0,0 +1,593 @@
+                || ApiBrowserConstants.check(url, ApiBrowserConstants.VIEW_BUNDLEGROUP)) {
+            point = ApiBrowserConstants.LIST_BUNDLEGROUPS;
+        } else if (ApiBrowserConstants.check(url, ApiBrowserConstants.LIST_BUNDLES)
+                || ApiBrowserConstants.check(url, ApiBrowserConstants.VIEW_BUNDLE)) {
+            point = ApiBrowserConstants.LIST_BUNDLES;
+        } else if (ApiBrowserConstants.check(url, ApiBrowserConstants.LIST_COMPONENTS)
+                || ApiBrowserConstants.check(url, ApiBrowserConstants.VIEW_COMPONENT)) {
+            point = ApiBrowserConstants.LIST_COMPONENTS;
+        } else if (ApiBrowserConstants.check(url, ApiBrowserConstants.LIST_SERVICES)
+                || ApiBrowserConstants.check(url, ApiBrowserConstants.VIEW_SERVICE)) {
+            point = ApiBrowserConstants.LIST_SERVICES;
+        } else if (ApiBrowserConstants.check(url, ApiBrowserConstants.LIST_EXTENSIONPOINTS)
+                || ApiBrowserConstants.check(url, ApiBrowserConstants.VIEW_EXTENSIONPOINT)) {
+            point = ApiBrowserConstants.LIST_EXTENSIONPOINTS;
+        } else if (ApiBrowserConstants.check(url, ApiBrowserConstants.LIST_CONTRIBUTIONS)
+                || ApiBrowserConstants.check(url, ApiBrowserConstants.VIEW_CONTRIBUTION)) {
+            point = ApiBrowserConstants.LIST_CONTRIBUTIONS;
+        } else if (ApiBrowserConstants.check(url, ApiBrowserConstants.LIST_OPERATIONS)
+                || ApiBrowserConstants.check(url, ApiBrowserConstants.VIEW_OPERATION)) {
+            point = ApiBrowserConstants.LIST_OPERATIONS;
+        } else if (ApiBrowserConstants.check(url, ApiBrowserConstants.VIEW_DOCUMENTATION)) {
+            point = ApiBrowserConstants.VIEW_DOCUMENTATION;
+        }
+        if (point == null) {
+            // check plugins
+            List<Plugin<?>> plugins = getSnapshotManager().getPlugins();
+            for (Plugin<?> plugin : plugins) {
+                point = plugin.getView(url);
+                if (point != null) {
+                    break;
+                }
+            }
+        }
+
+        return point;
+    }
+
+    @GET
+    @Produces("text/html")
+    public Object doGet() {
+        return getView("index").arg("hideNav", Boolean.TRUE);
+    }
+
+    @Path("latest")
+    public Resource getLatest() {
+        List<DistributionSnapshot> snaps = listPersistedDistributions();
+        Optional<DistributionSnapshot> distribution = snaps.stream()
+                                                           .filter(snap -> snap.getName()
+                                                                               .toLowerCase()
+                                                                               .startsWith("nuxeo platform"))
+                                                           .findFirst();
+
+        String latest = "current";
+        if (distribution.isPresent()) {
+            latest = distribution.get().getKey();
+        }
```

Comment:
Could be:
```java
String latest = snaps.stream()
                      .filter(snap -> snap.getName().toLowerCase().startsWith("nuxeo platform"))
                      .map(DistributionSnapshot::getKey)
                      .findFirst()
                      .orElse("current");
```

Reply:
indeed but i only moved that code -> i would prefer not modifying it to avoid unforeseen regressions in these logics